kube_metrics_server/datadog_checks/kube_metrics_server/kube_metrics_server.py

```diff
@@ -6,18 +6,23 @@
 
 COUNTERS = {
     'authenticated_user_requests': 'authenticated_user.requests',
-    'metrics_server_kubelet_summary_scrapes_total': 'kubelet_summary_scrapes_total',
+    'metrics_server_kubelet_summary_scrapes_total': 'kubelet_summary_scrapes_total', # < 0.4.0 metrics server
+    'metrics_server_kubelet_request_total': 'kubelet_summary_scrapes_total' # > 0.4.0 metrics server
 }
 
 HISTOGRAMS = {
     'metrics_server_manager_tick_duration_seconds': 'manager_tick_duration',
     'metrics_server_scraper_duration_seconds': 'scraper_duration',
 }
 
-SUMMARIES = {'metrics_server_kubelet_summary_request_duration_seconds': 'kubelet_summary_request_duration'}
+SUMMARIES = {
+    'metrics_server_kubelet_summary_request_duration_seconds': 'kubelet_summary_request_duration',  # < 0.4.0 metrics server
+    'metrics_server_kubelet_request_duration_seconds': 'kubelet_summary_request_duration' # > 0.4.0 metrics server
+}
 
 GAUGES = {
-    'metrics_server_scraper_last_time_seconds': 'scraper_last_time',
+    'metrics_server_scraper_last_time_seconds': 'scraper_last_time', # < 0.4.0 metrics server
+    'metrics_server_kubelet_last_request_time_seconds': 'scraper_last_time' # > 0.4.0 metrics server
```

Comment:
```suggestion
    'metrics_server_kubelet_last_request_time_seconds': 'scraper_last_time', # > 0.4.0 metrics server
```

^CI failing on missing `,`

Reply:
ah good catch! fixed